source/dsl/DangerDSL.ts

```diff
@@ -119,12 +122,23 @@ export interface DangerDSLType {
    *  comments and reviews on the PR, related issues, commits, comments
    *  and activities.
    *
-   *  Strictly speaking, `bitbucket_server` is a nullable type, if you are using
-   *  GitHub then it will be undefined. For the DSL convenience sake though, it
+   *  Strictly speaking, `bitbucket_server` is a nullable type, if you are not using
+   *  BitBucket Server then it will be undefined. For the DSL convenience sake though, it
    *  is classed as non-nullable
    */
   readonly bitbucket_server: BitBucketServerDSL
 
+  /**
+   * The GitLab metadata. This covers things like PR info,
+   * comments and reviews on the MR, commits, comments
+   * and activities.
+   *
+   * Strictly speaking, `gitlab` is a nullable type, if you are not using
+   * GitLab then it will be undefined. For the DSL convenience sake though, it
+   * is classed as non-nullable
+   */
+  readonly gitlab: GitLabDSL
```

Comment:
/cc @f-meloni - this is going to cause you trouble once this is released ;)

Reply:
It shouldn't I think, we won't parse it then until when there is no support it should be fine no?
I didn't look at the pr in detail though
And I think I will add the support for gitlab on danger-swift soon at this point 😁

Reply:
Ah yeah, great 👍